bench/telemetry/run.sh

```
@@ -0,0 +1,64 @@
+TCK=$(getconf CLK_TCK)
+out=$(mktemp -d)
+
+COLLECTOR_PORT=$COLLECTOR_PORT taskset -c 30 $BUN "$here/collector.js" 2>/dev/null & COL=$!
+trap 'kill $COL 2>/dev/null || true; rm -rf $out' EXIT
+sleep 0.3
+
+cputicks() { awk '{print $14+$15}' /proc/$1/stat; }
+
+runone() {
+  local key=$1 bin=$2; shift 2
+  env "$@" PORT=$PORT AWAITS=$AWAITS OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:$COLLECTOR_PORT nice -n -15 taskset -c $SERVER_CPU "$bin" "$here/server.js" 2>>$out/$key.err & local PID=$!
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)